src/cypress/integration/demo/trade.js

```diff
@@ -0,0 +1,99 @@
+import { Given, When, Then, And } from 'cypress-cucumber-preprocessor/steps'
+
+Given('Reactive Trader is open', () => {
+  cy.visit(Cypress.env('base_url'))
+})
+And('{string} price tile is displayed', symbol => {
+  cy.VerifyCurrencyOnPage(symbol)
+})
+
+When('user performs {string} trade for {string} of {string}', (direction, notional, symbol) => {
+  cy.performTrade(symbol, notional, direction)
+})
+
+Then('trade is success for {string} and {string}', (notional, direction) => {
+  cy.verifyTradeSuccess(notional, direction)
```

Comment:
I still don't like that there is some state held in the commands, when you read this file you have no idea that is happening behind the scenes and the command file is hard to read and follow code.

At a minimum I think the state should be held here, to keep each command isolated and to give the control the tests. 

Example - 

```
let symbol;

When('user performs {string} trade for {string} of {string}', (direction, notional, symbol) => {
  symbol = symbol
  cy.performTrade(symbol, notional, direction)
})

Then('trade is success for {string} and {string}', (notional, direction) => {
  cy.verifyTradeSuccess(notional, direction, symbol)
})
```

Alternatively (and perhaps more appropriate) you could look into using sharing context - https://github.com/TheBrainFamily/cypress-cucumber-preprocessor#sharing-context